otelcollector/build/windows/Dockerfile

```diff
@@ -36,40 +35,15 @@ COPY ./telegraf/telegraf-prometheus-collector-windows.conf $tmpdir/telegraf/
 COPY ./fluent-bit/fluent-bit-windows.conf $tmpdir/fluent-bit/
 COPY ./fluent-bit/fluent-bit-parsers.conf $tmpdir/fluent-bit/
 COPY ./fluent-bit/src/out_appinsights.so $tmpdir/fluent-bit/bin/
+COPY ./main/main.exe $tmpdir/microsoft
 COPY ./react /Users/ContainerAdministrator/Documents/static/react
 COPY ./LICENSE $tmpdir/microsoft
 COPY ./NOTICE $tmpdir/microsoft
 
-# Do not split this into multiple RUN!
-# Docker creates a layer for every RUN-Statement
-ENV chocolateyVersion "1.4.0"
-RUN powershell -Command "Set-ExecutionPolicy Bypass -Scope Process -Force; iex ((New-Object System.Net.WebClient).DownloadString('https://chocolatey.org/install.ps1'))"
-# Fluentd depends on cool.io whose fat gem is only available for Ruby < 2.5, so need to specify --platform ruby when install Ruby > 2.5 and install msys2 to get dev tools
-RUN choco install -y ruby --version 2.6.5.1 --params "'/InstallDir:C:\ruby26'" \
-&& choco install -y msys2 --version 20211130.0.0 --params "'/NoPath /NoUpdate /InstallDir:C:\ruby26\msys64'" \
-&& choco install -y vim
-
-# gangams - optional MSYS2 update via ridk failing in merged docker file so skipping that since we dont need optional update
-RUN refreshenv \
-&& ridk install 3 \
-&& echo gem: --no-document >> C:\ProgramData\gemrc \
-&& gem install cool.io -v 1.5.4 --platform ruby \
-&& gem install oj -v 3.3.10 \
-&& gem install json -v 2.2.0 \
-&& gem install fluentd -v 1.12.2 \
-&& gem install win32-service -v 1.0.1 \
-&& gem install win32-ipc -v 0.7.0 \
-&& gem install win32-event -v 0.6.3 \
-&& gem install windows-pr -v 1.2.6 \
-&& gem install tomlrb -v 1.3.0 \
-&& gem install deep_merge -v 1.2.1\
-&& gem install colorize\
-&& gem sources --clear-all
-
```

Comment:
Do macro based switching for process_utilities for linux versus windows build

Reply:
Golang doesn't have the same macro based switching like ruby. We have to create two separate files from what I can see so I'm going forward with the docker ignore method here